### ISSUE TITLE
Copy changes UTF8 text file encoding to ASCII on Windows

### ISSUE THREAD

**Issue by bbaia** [gulpjs/gulp#1546]
Hi,

A gulp copy of a robots.txt file, changes the encoding from UTF8 to ASCII.

```javascript
return gulp.src(['robots.txt'])
           .pipe(gulp.dest('build'));
```

Is this a bug or an expected behavior ?

TIA,
Bruno

**Comment by phated** [comment]
I'm pretty sure we are writing files with utf8 encoding.  What version of gulp, etc are you using?

**Comment by bbaia** [comment]
gulp v3.9.1
node v4.3.1

**Comment by phated** [comment]
Can you try this in gulp 4?

**Comment by phated** [comment]
I'm pretty sure we aren't writing these files as ASCII.  How did you determine this?

**Comment by roselan** [comment]
I believe  have this issue too when trying to build icons for https://github.com/swisnl/jQuery-contextMenu.

By example, "delete" icon character becomes "EE A8 84" instead of "EE AA BC".

**Comment by roselan** [comment]
Actually, it was linked to https://github.com/nfroidure/gulp-iconfont/issues/64 . sorry sorry!

**Comment by phated** [comment]
@roselan alright, glad you figured it out

@bbaia I'm going to close due to inactivity.  I have no idea how to follow up on this without more information.

**Comment by ftanrisevdi** [comment]
@bbaia i had the same issue but i upgrade my node and gulp. It is ok now. 

```
gulp.task('copy', function () {
    return gulp.src(['Scripts/app/**/*.html'])
      .pipe(gulp.dest('./dist/js'));
});
```

**Comment by patrickbussmann** [comment]
@ftanrisevdi and @phated:
I tried with `node` 6.11.3 and `gulp` 4.0.0-alpha2 and it not work.
The files which are written in `UTF-8-BOM` will be `System` (ASCII) afterwards.

I also tried the https://www.npmjs.com/package/gulp-convert-encoding plugin which not works, too.

    .pipe(convertEncoding({to: 'utf8'}))

And the https://www.npmjs.com/package/gulp-charset plugin.

    .pipe(charset({to: 'UTF-8'}))

But nothing works 😭
